app/channels/solution_channel.rb

```diff
@@ -10,6 +10,7 @@ def unsubscribed; end
   def self.broadcast!(solution)
```

Comment:
@iHiD I didn't know where to put the webhook for the iterations. I just put this here for now.